app/controllers/case_workers/claims_controller.rb

```diff
@@ -26,6 +26,17 @@ def archived; end
 
     def show
       prepare_show_action
+      @active_tab = params[:tab] || 'information'
+    end
+
+    def status
+      prepare_show_action
+      render 'case_workers/claims/claim_status'
+    end
```

Comment:
Is this action (and the `information` action below) being used? It looks as though `show` is being used and the `tab` parameter determines which page to show. I think this is preferable to having two extra actions and corresponding endpoints.